README.md

```diff
@@ -1,4 +1,6 @@
 # SwiftCuckoo
+![Build](https://github.com/tavernari/swiftcuckoo/actions/workflows/swift.yml/badge.svg)
+[![Maintainability](https://api.codeclimate.com/v1/badges/315d2f68bad03d358e42/maintainability)](https://codeclimate.com/github/Tavernari/SwiftCuckoo/maintainability)
```

Comment:
ℹ️ Add badges to the README like this:

<img width="863" alt="Screenshot 2024-09-03 at 05 55 05" src="https://github.com/user-attachments/assets/baa1339f-042c-4749-bcea-ae35cd45af08">